### DISCUSSION TITLE
Push Notification

### DISCUSSION THREAD

**Discussion by ankwad** [chatwoot/chatwoot#5653] [Self hosted]
Can anyone help with Push notification
it doesnt work for new messages for an assigned conversation
also if a team is assigned to a conversation .- why doesnt the team get notification for the new messages
Whats the logic of getting push notifications

Also No notifications come for an API chanel - I have assigned the chanel conversation to an agent - still they dont get any notification

**Comment by sadden3194** [comment]
Hi!

which browser? 
I have the same problem with firefox and safari. With google chrome it works
